ui/src/cohortToolbar.tsx

```diff
@@ -1,19 +1,38 @@
 import RedoIcon from "@mui/icons-material/Redo";
 import UndoIcon from "@mui/icons-material/Undo";
 import Button from "@mui/material/Button";
+import Divider from "@mui/material/Divider";
 import Stack from "@mui/material/Stack";
-import { useAppDispatch, useAppSelector, useUndoRedoUrls } from "hooks";
+import {
+  useAppDispatch,
+  useAppSelector,
+  useCohort,
+  useUnderlay,
+  useUndoRedoUrls,
+} from "hooks";
 import { Link as RouterLink } from "react-router-dom";
 import { ActionCreators as UndoActionCreators } from "redux-undo";
+import { cohortReviewURL } from "router";
+
+export default function CohortToolbar() {
+  const underlay = useUnderlay();
+  const cohort = useCohort();
 
-function UndoRedo() {
```

Comment:
Why did you remove the `UndoRedo` toolbar function? I see that it's replaced with `CohortToolbar` in the other files as well. Is it just always included everywhere the `Cohorts` toolbar button is so you bundled them?

Reply:
Yeah, I need to specify the same set of buttons for the `ActionBar` everywhere within the cohort editor so it's easiest to group them within a component. I could have made a new one that also included the `UndoRedo` component but it's also only used within the cohort editor and I don't expect it to be used elsewhere so it made more sense to just merge them together.